### DISCUSSION TITLE
I use Pushy to push the message,then I get "success=true",but the phone not responding

### DISCUSSION THREAD

**Discussion by woTMshuaideyimapi** [relayrides/pushy#1031] [Q&A]
I setApnsServer(ApnsClientBuilder.DEVELOPMENT_APNS_HOST) and get the response "success=true",,but the phone not responding ,why it happen

**Comment by jchambers** [comment]
This does not sound like a bug with Pushy. By all appearances, Pushy is communicating with the APNs server and reporting the response from the server as expected. With that in mind, I'm going to convert this from an issue (issues imply that there's something to fix within Pushy itself) to a discussion (a more appropriate place for general support questions).

**Comment by woTMshuaideyimapi** [comment]
There is a strange phenomenon here.I setApnsServer(ApnsClientBuilder.DEVELOPMENT_APNS_HOST) and get the response "success=true",,but the phone not responding.But I use the tool(This tool is called smartpush) to send the same parameters, the URL pushed by this tool is "https://api.sandbox.push.apple.com/3/device/<DeviceToken>",And then the device received the message.Can you provide me some suggestions?
